pkg/pillar/devicenetwork/dnc.go

```diff
@@ -310,6 +323,12 @@ func VerifyPending(pending *DPCPending,
 		errStr := fmt.Sprintf("Failed network test: %s",
 			err)
 		log.Errorf("VerifyPending: %s\n", errStr)
+		if cf {
+			log.Errorf("VerifyPending: certificate failure")
+			// NOTE: do not increase TestCount; we retry until
+			// the certificate is fixed on the server side.
+			status = DPC_WAIT
```

Comment:
When certificate related failures happen, are we sending an error as part of device status? Does it make sense to send something like a "Cloud certificate error" as part of device status with the last try timestamp to cloud. This can be an alert for someone to look for fix.

Reply:
Can't send anything since we can't connect to the cloud at that point in time. All we can do is to more clearly make it appear in the diag output.

Reply:
Right.